mlflow/server/handlers.py

```diff
@@ -611,12 +611,8 @@ def wrapper(*args, **kwargs):
 
 @catch_mlflow_exception
 def get_artifact_handler():
-    from querystring_parser import parser
-
-    query_string = request.query_string.decode("utf-8")
-    request_dict = parser.parse(query_string, normalized=True)
-    run_id = request_dict.get("run_id") or request_dict.get("run_uuid")
-    path = request_dict["path"]
+    run_id = request.args.get("run_id") or request.args.get("run_uuid")
+    path = request.args["path"]
```

Comment:
The query string for `get-artifact` looks like this:

```
http://127.0.0.1:7000/get-artifact?path=model%2FMLmodel&run_uuid=60549ecdac2d4b1391001fc15faede47
```

It's a simple query string.

Reply:
I manually confirmed UI works fine with this change.

<img width="725" alt="image" src="https://github.com/user-attachments/assets/bef254d1-0cfc-4bd4-88e0-88429aac045a">

Reply:
The same comment on other two handlers.